WeRockIT-ui/.eslintrc.js

```diff
@@ -3,10 +3,28 @@ module.exports = {
   env: {
     node: true
   },
-  extends: ["plugin:vue/essential", "@vue/prettier"],
+  extends: [
+    "plugin:vue/essential",
+    "plugin:prettier/recommended",   // turns off all ESLINT rules that are unnecessary due to Prettier or might conflict with Prettier. 
+    "@vue/prettier"
+  ],
+  plugins: ['vue', 'prettier'],
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
-    "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off"
+    "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",
+    "vue/max-attributes-per-line": "off",
```

Comment:
You have `singleQuote` set to true below but still have all of these as double quotes. Is there a reason they couldn't be changed?

Reply:
Nope.  Just me not thinking about it.  I think I copied the `singleQuote` attribute from the prettier config we originally had.  Let me go through and swap this to be single quotes. :3